app/controllers/general.php

```diff
@@ -157,6 +158,9 @@
     $responseFormat = $request->getHeader('x-appwrite-response-format', App::getEnv('_APP_SYSTEM_RESPONSE_FORMAT', ''));
     if ($responseFormat) {
         switch($responseFormat) {
+            case version_compare ($responseFormat , '0.12.3', '<='):
```

Comment:
Why this specific version? Cant we do `0.12.0` or `0.13.0`?

Reply:
the version_compare should compare those versions automatically

Reply:
We released 0.12.4. Will that work fine? That's the only concern I have.

Reply:
we should give the number of our latest version here. 
so @PineappleIOnic , we need to replace it with `0.12.4`

Reply:
We should do `0.12.*` - so we don't have to care about this.

Reply:
nvm, not working

Reply:
I will either sound really smart or really dumb here, but why not `'0.13.0', '<'`?